lake/pool.go

```diff
@@ -239,11 +242,22 @@ func (p *Pool) StoreInStaging(ctx context.Context, txn *commit.Transaction) erro
 	return storage.Put(ctx, p.engine, p.StagingObject(txn.ID), bytes.NewReader(b))
 }
 
-func (p *Pool) ScanStaging(ctx context.Context, w zio.Writer, ids []ksuid.KSUID) error {
+// ScanStaging writes the provided commits in staging to the provided
+// zio.Writer.  If not commit tags are passed, all commits in staging are
+// written.
+func (p *Pool) ScanStaging(ctx context.Context, w zio.Writer, ids []ksuid.KSUID) (err error) {
+	if len(ids) == 0 {
+		ids, err = p.ListStagedCommits(ctx)
```

Comment:
NIt: I think it's clearer to use an explicit declaration here. The named result parameter suggests something more complicated is going on.
```suggestion
func (p *Pool) ScanStaging(ctx context.Context, w zio.Writer, ids []ksuid.KSUID) error {
	if len(ids) == 0 {
		var err error
		ids, err = p.ListStagedCommits(ctx)
```

Reply:
Sure!